content/en/blog/news/2022-gsoc-followup.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Google Summer of Code 2022 ☀️"
+date: 2022-03-12
+draft: false
+description: >
+  Jenkins X GSoC 2022 followup!
+categories: [blog]
+keywords: [Community, 2022]
+slug: "GSoC2022-followup"
+aliases: []
+author: Ankit D Mohapatra
+---
+
+We are very happy to announce that Jenkins X has been selected to participate in the Google Summer of Code (GSoC) 2022!
+
+If you are new to GSoC and want to learn more about it, check out their new [site](https://summerofcode.withgoogle.com/).
+You can find the list of project ideas from Jenkins X [here](/blog/2022/02/20/gsoc2022-ideas/).
+If you are interested in applying to Jenkins X, please check the [timeline](https://developers.google.com/open-source/gsoc/timeline).
+Apply as early as possible.
+
+This post aims to give GSoC contributors insight into the selection process and how to get started.
+We want to be transparent about the entire process.
```

Comment:
Great intention - "We want to be transparent about the entire process." 
But you may want to take out the sentence in this post because this intention is for your GSoC org team, in terms of goals in how to execute and communicate. The GSoC applicants don't need to keep this thought in mind -- there's a lot of other information they need to prioritise. :)

Reply:
Makes sense, removing.